test_vgpu.sh

```diff
@@ -31,10 +31,13 @@ TEMPEST_DIR=$DEST/tempest
 NOVA_CONF=/etc/nova/nova.conf
 IMAGE_NAME=${IMAGE_NAME:-"cirros-0.3.5-x86_64-disk"}
 VM_NAME=${VM_NAME:-"testVM"}
+SNAPSHOT_NAME=${SNAPSHOT_NAME:-"testSnapshot"}
 JOURNAL_DIR=/var/log/journal
-VGPU_TEST_LOG_DIR=${VGPU_TEST_LOG_DIR:-"/opt/stack/workspace/test_vgpu/logs"}
+OPENSTACK_LOGS=${OPENSTACK_LOGS:-"/opt/stack/workspace/test_vgpu/logs"}
 TMP_LOG_DIR=/tmp/openstack
 
+VGPU_CREATE_SUCCESS_FLAG="VGPU_CREATE_SUCCESS"
```

Comment:
Rather than these we should have a separate PASSED (and probably FAILED flag).
Is a different script grepping for the result?  If so, then we can easily grep for FAILED.
Perhaps this script should be wrapped so we can treat any failure as FAILED (i.e. the test_vgpu.sh script has "set -ex" at the top, but the calling script will consider the script's exit code)

Reply:
Got it, will modify.

Reply:
done